### PR TITLE
Fix conformance test and update namespace

### PR DESCRIPTION
**Pull request by shubham1172**
Signed-off-by: Shubham Sharma <shubhash@microsoft.com>

# Description

After recent infra migration, namespace should move from `dapr2` to `dapr3`

## Issue reference

Please reference the issue this PR will close: #1637

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/certification/secretstores/azure/keyvault/managed-identity-app/Dockerfile [hunk 1]
```diff
@@ -4,5 +4,5 @@ WORKDIR /app
 RUN apt-get update && apt-get install wget curl --yes
 RUN wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
 RUN dapr init --slim
-ENV AzureKeyVaultName=dapr2-conf-test-kv
+ENV AzureKeyVaultName=dapr3-conf-test-kv
```
Comment (surenderssm) [inline tests/certification/secretstores/azure/keyvault/managed-identity-app/Dockerfile:7]: Is this also leading to any failures?

**Comment by surenderssm** [comment]
Looks good to me.

File: tests/certification/secretstores/azure/keyvault/managed-identity-app/Dockerfile (hunk 1 above)
Comment (shubham1172) [reply tests/certification/secretstores/azure/keyvault/managed-identity-app/Dockerfile:7]: Not that I see any!

File: tests/config/pubsub/tests.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ components:
     allOperations: true
     config:
       pubsubName: azure-eventhubs
-      testTopicName: dapr2-conf-test-eventhubs-pubsub-topic
+      testTopicName: dapr3-conf-test-eventhubs-pubsub-topic
```
[the remark this answers is not in the document]
Comment (shubham1172) [reply tests/config/pubsub/tests.yml:16]: Thanks, created one: https://github.com/dapr/components-contrib/issues/1641

**Comment by berndverst** [comment]
I already fixed this in release branch last week -- I shared that with the team in dapr core chat. Will be fixed when release is merged into master.

**Comment by berndverst** [comment]
https://github.com/dapr/components-contrib/pull/1642

File: tests/certification/secretstores/azure/keyvault/managed-identity-app/Dockerfile (hunk 1 above)
Comment (berndverst) [reply tests/certification/secretstores/azure/keyvault/managed-identity-app/Dockerfile:7]: This is no longer used.. I will be removing this test.

**Pull request closed without merging**